flink-ai-flow/lib/airflow/airflow/migrations/versions/d47076ca1426_add_event_based_scheduler_support.py

```diff
@@ -59,6 +59,11 @@ def upgrade():
         batch_op.add_column(sa.Column('event_ack_id', sa.BigInteger(), nullable=True))
         batch_op.add_column(sa.Column('context', sa.Text(), nullable=True))
 
+    """Apply Add is_active Column to task_instance table"""
```

Comment:
It may be not friendly for users to upgrade if written here as we have published a stable version, it would be better to write DDL to `fb02e046d5e6_add_task_execution_label.py` so that we can tell users how to upgrade DDL in the release note.